test/test_issue325.js

```diff
@@ -0,0 +1,43 @@
+
+exports.test = function(sql, assert){
+    "use strict";
+    // Create a database
+    var db = new sql.Database();
+    var result;
+    var value;
+
+    // inline result value test
+    result = db.exec("SELECT 1.7976931348623157e+308 AS value");
+    value = result[0].values[0][0];
+    assert.strictEqual(value, 1.7976931348623157e+308);
+
+    // bind result value test
+    result = db.exec("SELECT ? AS value", [1.7976931348623157e+308]);
+    value = result[0].values[0][0];
+    assert.strictEqual(value, 1.7976931348623157e+308);
+
+    // CAST value test
+    result = db.exec("SELECT CAST('1.7976931348623157e+308' AS REAL) AS value");
+    value = result[0].values[0][0];
+    assert.strictEqual(value, 1.7976931348623157e+308);
```

Comment:
Can you please add a message to your assertions ? Currently, they appear in the tests as 

```
   test issue325
    ✓ undefined
    ✓ undefined
    ✓ undefined
```

Reply:
added messsages in force-commit 4a635195
```
  test issue325
    ✓ SELECT 1.7976931348623159e+308 is Infinity
    ✓ SELECT 1.7976931348623159e+308 is Infinity
    ✓ SELECT 1.7976931348623157e+308 is not Infinity
    ✓ SELECT 1.7976931348623157e+308 is 1.7976931348623157e+308
    ✓ SELECT ? AS value is not Infinity
    ✓ SELECT ? AS value is 1.7976931348623157e+308
    ✓ SELECT CAST('1.7976931348623157e+308' AS REAL) is not Infinity
    ✓ SELECT CAST('1.7976931348623157e+308' AS REAL) is 1.7976931348623157e+308
```